### PR TITLE
Cvs 30080 ovms quickstart short

### PR DESCRIPTION
**Pull request by jcchr**

### CODE REVIEW

File: docs/ovms_quickstart.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+└── face-detection
+    └── 1
+        ├── face-detection-0106.bin
+        └── face-detection-0106.xml
+
+``` 
+
+More about this structure and about how to deploy more than one model at the time can be found [here](./docker_container.md#preparing-the-models) and [here](./docker_container.md#starting-docker-container-with-a-configuration-file).
+
+### Download the example client script
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:52]: Please change to `### Download the Example Client Script`

File: docs/ovms_quickstart.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+can be downloaded from various sites in the Internet (for example from the [Open Model Zoo](https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/) ) or converted from other formats - like TensorFlow, ONNX, Caffe or MXNet using [Model Optimizer](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html) which is a part of the OpenVINO Toolkit.
+
+Let's use the face detection model available under the following [link](https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/). 
+
+Then download both components of the model to the `model` directory (for example using curl):
+```
+curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/face-detection-retail-0004.xml -o model/face-detection-retail-0004.bin
+```
+
+### Start the Model Server container
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:30]: Change to `### Start the Model Server Container`

File: docs/ovms_quickstart.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+There are also other ways to obtain Model Server application:
+* Building a container image with a Dockerfile - more details can be found [here](docker_container.md)
+* Building the application locally from source - detailed description is located [here](host.md)  
+
+### Download a desired model
+
+OpenVino&trade; Model Server requires a trained model to be able to perform an inference. The model
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:19]: Change `OpenVino&trade;` to `OpenVINO`

File: docs/ovms_quickstart.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+following command to start using it:
+
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+There are also other ways to obtain Model Server application:
+* Building a container image with a Dockerfile - more details can be found [here](docker_container.md)
+* Building the application locally from source - detailed description is located [here](host.md)  
+
+### Download a desired model
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:17]: Change to `### Download a Model`

File: docs/ovms_quickstart.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+# OpenVINO&trade; Model Server Quickstart
+
+To quickly start using OpenVino&trade; Model Server please execute the following steps.
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:3]: Change `OpenVino&trade;` to `OpenVINO`

File: docs/ovms_quickstart.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+```
+curl https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_utils.py -o client_utils.py https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/face_detection.py -o face_detection.py  https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_requirements.txt -o client_requirements.txt
+```
+
+More details about this particular script can be found [here](../example_client/face_detection.md). There is more scripts client scripts provided for Model Server - the
+full list is [here](../example_client). 
+
+
+### Download the data needed for inference
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:68]: Change to `### Download Data for Inference`

File: docs/ovms_quickstart.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+Example images that can be used for inference can be downloaded from this [url](../example_client/images/people). Let's download the following [one](../example_client/images/people/people1.jpeg). The image should be located in the separate folder - as the script we are going to use makes an inference on all images in
+a folder passed to it as a parameter.
+
+```
+curl --create-dirs https://github.com/openvinotoolkit/model_server/blob/master/example_client/images/people/people1.jpeg?raw=true -o images/people1.jpeg
+```
+
+### Run the inference
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:77]: Change to `Run Inference`

File: README.md [hunk 8]
```diff
@@ -17,9 +17,40 @@ A few key features:
 
 ## Getting Up and Running
 
-[Using a docker container](docs/docker_container.md)
+Start using OpenVino&trade; Model Server in 5 Minutes or less
 
-[Landing on bare metal or virtual machine](docs/host.md)
+```bash
+# Download the OpenVino Model Server image
```
Comment (ryanloney) [inline README.md:23]: Change to `# Download the latest Model Server image`

File: README.md [hunk 9]
```diff
@@ -17,9 +17,40 @@ A few key features:
 
 ## Getting Up and Running
```
Comment (ryanloney) [inline README.md:18]: Change to `## Running the Server`

File: README.md [hunk 10]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
 
-[Using a docker container](docs/docker_container.md)
+Start using OpenVino&trade; Model Server in 5 Minutes or less
```
Comment (ryanloney) [inline README.md:20]: Change to `Start using OpenVINO Model Server in 5 Minutes or less:`

File: README.md [hunk 11]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
 
-[Landing on bare metal or virtual machine](docs/host.md)
+```bash
+# Download the OpenVino Model Server image
+docker pull openvino/ubuntu18_model_server:latest
+
+# Download model into a separate directory
+curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/face-detection-retail-0004.xml -o model/face-detection-retail-0004.bin
+
+# Start the Model Server container serving the gRpc port
```
Comment (ryanloney) [inline README.md:29]: Change to `# Start the Model Server container serving gRPC on port 9000

File: README.md [hunk 12]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+docker run -d -v $(pwd)/model:/models/face-detection/1 -e LOG_LEVEL=DEBUG -p 9000:9000 openvino/ubuntu18_model_server /ie-serving-py/start_server.sh ie_serving model --model_path /models/face-detection --model_name face-detection --port 9000  --shape auto
+
+# Download the example client script
+curl https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_utils.py -o client_utils.py https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/face_detection.py -o face_detection.py  https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_requirements.txt -o client_requirements.txt
+
+# Download and image to be analyzed
```
Comment (ryanloney) [inline README.md:35]: Change to `# Download an image to be analyzed`

File: README.md [hunk 13]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+curl --create-dirs https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/images/people/people1.jpeg -o images/people1.jpeg
+
+# Install client's dependencies
```
Comment (ryanloney) [inline README.md:38]: Change to `# Install client dependencies`

File: README.md [hunk 14]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+pip install -r client_requirements.txt
+
+# Create folder for results
```
Comment (ryanloney) [inline README.md:41]: Change to `# Create a folder for results`

File: README.md [hunk 15]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+mkdir results
+
+# Run the inference and store its results in the newly created folder
```
Comment (ryanloney) [inline README.md:44]: Change to `# Run inference and store results in the newly created folder`

File: README.md [hunk 16]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+python face_detection.py --batch_size 1 --width 600 --height 400 --input_images_dir images --output_dir results
+```
+The more detailed description (however still quick) of the steps above can be found [here](docs/ovms_quickstart.md).
```
Comment (ryanloney) [inline README.md:47]: Change to `A more detailed description of the steps above can be found [here](docs/ovms_quickstart.md).`

File: README.md [hunk 17]
```diff
@@ -17,9 +17,40 @@ A few key features:
...
+
+Comprehensive description of how to use Model Server in various scenarios can be found in the two following links:
```
Comment (ryanloney) [inline README.md:49]: Change to `More complete guides to using Model Server in various scenarios can be found here:`

File: docs/ovms_quickstart.md [hunk 18]
```diff
@@ -0,0 +1,101 @@
+# OpenVINO&trade; Model Server Quickstart
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:1]: Change `OpenVino&trade;` to `OpenVINO`

File: docs/ovms_quickstart.md [hunk 19]
```diff
@@ -0,0 +1,101 @@
+# OpenVINO&trade; Model Server Quickstart
+
+To quickly start using OpenVino&trade; Model Server please execute the following steps.
+
+### Deploy Model Server
+
+The Docker image with OpenVino&trade; Model Server is available on the DockerHub. Issue the
+following command to start using it:
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:8]: Change to `A Docker image with OpenVINO Model Server is available on DockerHub. Use the following command to download the latest release:`

File: docs/ovms_quickstart.md [hunk 20]
```diff
@@ -0,0 +1,101 @@
...
+
+The Docker image with OpenVino&trade; Model Server is available on the DockerHub. Issue the
+following command to start using it:
+
+```bash
+docker pull openvino/ubuntu18_model_server
+```
+There are also other ways to obtain Model Server application:
+* Building a container image with a Dockerfile - more details can be found [here](docker_container.md)
+* Building the application locally from source - detailed description is located [here](host.md)
```
Comment (ryanloney) [inline docs/ovms_quickstart.md:15]: Change to 
```
In addition to the pre-built container image, see instructions to:
* [Build a container image with a Dockerfile](docker_container.md)
* [Build the application locally from source](host.md)
```

File: docs/ovms_quickstart.md (hunk 1 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:52]: corrected

File: docs/ovms_quickstart.md (hunk 2 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:30]: corrected

File: docs/ovms_quickstart.md (hunk 3 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:19]: done

File: docs/ovms_quickstart.md (hunk 4 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:17]: done

File: docs/ovms_quickstart.md (hunk 5 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:3]: done

File: docs/ovms_quickstart.md (hunk 6 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:68]: done

File: docs/ovms_quickstart.md (hunk 7 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:77]: done

File: README.md (hunk 8 above)
Comment (jcchr) [reply README.md:23]: done

File: README.md (hunk 9 above)
Comment (jcchr) [reply README.md:18]: done

File: README.md (hunk 10 above)
Comment (jcchr) [reply README.md:20]: done

File: README.md (hunk 11 above)
Comment (jcchr) [reply README.md:29]: done

File: README.md (hunk 12 above)
Comment (jcchr) [reply README.md:35]: corrected

File: README.md (hunk 13 above)
Comment (jcchr) [reply README.md:38]: done

File: README.md (hunk 14 above)
Comment (jcchr) [reply README.md:41]: done

File: README.md (hunk 15 above)
Comment (jcchr) [reply README.md:44]: done

File: README.md (hunk 16 above)
Comment (jcchr) [reply README.md:47]: done

File: README.md (hunk 17 above)
Comment (jcchr) [reply README.md:49]: corrected

File: docs/ovms_quickstart.md (hunk 18 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:1]: done

File: docs/ovms_quickstart.md (hunk 19 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:8]: done

File: docs/ovms_quickstart.md (hunk 20 above)
Comment (jcchr) [reply docs/ovms_quickstart.md:15]: done

**Pull request merged**